### DISCUSSION TITLE
Multiple STOCK places

### DISCUSSION THREAD

**Discussion by wmdhosting** [craftcms/commerce#2669] [Ideas] [answered]
Most companies with multiple stores need tracking of products stock per locations..
Seen some other  newer commerce platforms.. having that by default..  too  
example.  https://saleor.io/

posiblity to define multiple STOCK location..

Multi-warehouse inventory
Track stock levels across multiple locations. Edit multiple products simultaneously. Check all out-of-stock items in seconds.

**Comment by nfourtythree** [comment, marked as answer]
Hi @wmdhosting 

This is a duplicate of and will be handled by #2286 

Thanks!
